funnel/models/session.py

```diff
@@ -114,6 +120,22 @@ def scheduled(self):
     def scheduled(self):
         return (self.start_at != None) & (self.end_at != None)  # NOQA
 
+    @cached_property
+    def start_at_localized(self):
+        return (
+            localize_timezone(self.start_at, tz=self.project.timezone)
+            if self.start_at and self.scheduled
+            else None
+        )
+
+    @cached_property
+    def end_at_localized(self):
+        return (
+            localize_timezone(self.end_at, tz=self.project.timezone)
+            if self.end_at and self.scheduled
+            else None
+        )
+
```

Comment:
You shouldn't be basing this on `self.scheduled`. Use just the datetime.

`self.scheduled` anyway just checks if there's a start/end date. Further, there's a SQL CHECK condition to ensure `start_at` and `end_at` are either both set or neither is set (a `NOT XOR` or `XNOR` condition), so there is no need to check `self.scheduled` at all.